### DISCUSSION TITLE
How to use date_sub in mysql

### DISCUSSION THREAD

**Discussion by 5000batt** [hasura/graphql-engine#8980] [Question] [answered]
END_DT >= DATE_SUB(NOW(), INTERVAL 10 MINUTE)

I am trying to use the corresponding phrase in mysql, how can I use it?

**Comment by adas98012** [comment, marked as answer]
Here is the MySql documents on usage of various Date-Time functions:

https://dev.mysql.com/doc/refman/8.0/en/date-and-time-functions.html

Hope, you find it useful for your case.

**Reply by 5000batt** [reply]
Thank you for answer.

It was a question about how to use it in hasura graphql, not how to use it in mysql.
